src/screens/CreatePost/elements/SheetMedia.tsx

```diff
@@ -70,11 +70,7 @@ const List = ({icon, label, onPress, isLoading = false}: ListProps) => (
   <TouchableOpacity style={styles.list} onPress={onPress}>
     {icon}
     <Text style={styles.labelList}>{label}</Text>
-    {isLoading ? (
-      <ActivityIndicator size="small" color={COLORS.darkBlue} style={styles.loader} />
-    ) : (
-      <></>
-    )}
+    {isLoading ? <ActivityIndicator size="small" color="#0000ff" style={styles.loader} /> : <></>}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded color value `#0000ff` should be replaced with a variable from the colors utility file for consistency and maintainability.
```diff
- {isLoading ? <ActivityIndicator size="small" color="#0000ff" style={styles.loader} /> : <></>}
+ {isLoading ? <ActivityIndicator size="small" color={colors.darkBlue} style={styles.loader} /> : <></>}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->